### PR TITLE
Remove deprecation warnings from docs pages

### PR DESCRIPTION
**Pull request by nvaytet**

### CODE REVIEW

File: docs/tutorials/1_exploring-data.ipynb [hunk 1]
```diff
@@ -503,7 +504,7 @@
     "- Inspect and understand the HTML view of the result.\n",
     "- Plot the result.\n",
     "  There are two options:\n",
-    "  - Use `plot` with `projection='1d'`\n",
+    "  - Use Plopp's `slicer` utility to navigate the different layers using a slider (requires `%matplotlib widget` to enable interactive figures)\n",
```
Comment (jl-wynen) [inline docs/tutorials/1_exploring-data.ipynb:507]: Add a link to 'slicer'?

File: docs/tutorials/2_working-with-masks.ipynb [hunk 2]
```diff
@@ -128,7 +127,7 @@
    },
    "outputs": [],
    "source": [
-    "data.plot()"
+    "data.hist(spectrum=500).transpose().plot()"
```
Comment (jl-wynen) [inline docs/tutorials/2_working-with-masks.ipynb:130]: Why does this need to be transposed now?

File: docs/tutorials/3_understanding-event-data.ipynb [hunk 3]
```diff
@@ -328,7 +328,7 @@
    },
    "outputs": [],
    "source": [
-    "tof_edges = sc.linspace(dim='tof', start=2000.0, stop=15000, num=200, unit='us')\n",
+    "tof_edges = sc.linspace(dim='tof', start=2000.0, stop=15000, num=201, unit='us')\n",
```
Comment (jl-wynen) [inline docs/tutorials/3_understanding-event-data.ipynb:331]: Good find :+1:

File: docs/tutorials/2_working-with-masks.ipynb (hunk 2 above)
Comment (nvaytet) [reply docs/tutorials/2_working-with-masks.ipynb:130]: The data has dims `(spectrum: 114688, tof: 200)`. The `spectrum` is too large to be plotted.
Doing `data.hist(spectrum=500)` yields dimensions `(tof: 200, spectrum: 500)` (dimension order changes).
So I need to transpose afterwards to get the tof axis along the horizontal direction.

**Review by jl-wynen — APPROVED** [review]

**Pull request merged**